src/html/template.html

```diff
@@ -1,5 +1,13 @@
-<div id="fl-mapviz-disclaimer">
-    Please note that this map is for demonstration purposes only and may not accurately reflect the accessibility or location of all COVID-19 test centres. To learn about the location of COVID-19 test centres, please visit your local health unit web site.
+<div id="fl-mapviz-provenance">
```

Comment:
With the change of the class name, [the definition](https://github.com/amb26/covid-data-monitor/blob/show-provenance/src/css/scss/fluid-covid-map-viz.scss#L637-L640) for the old class on the desktop view should be changed too.